Project.toml

```diff
@@ -2,6 +2,7 @@ name = "TextAnalysis"
 uuid = "a2db99b7-8b79-58f8-94bf-bbc811eef33d"
 license = "MIT"
 desc = "Julia package for text analysis"
+version = "0.6.1"
```

Comment:
This package hasnt been released in quite a while. So the next release has to be 0.7.0

Reply:
Done.